apps/jubmoji-quest/src/pages/collect/index.tsx

```diff
@@ -18,8 +18,8 @@ import { Onboarding } from "@/components/Onboarding";
 import {
   JubmojiCardProps,
   getJubmojiCardByPubIndex,
-  useCards,
-} from "@/hooks/useCards";
+  useFetchCards,
+} from "@/hooks/useFetchCards";
 import Image from "next/image";
 
 const OnboardSection = () => {
```

Comment:
Closing onboarding just leads to blank screen. 

![IMG_6838](https://github.com/jubmoji/jubmoji.quest/assets/6290822/9a439dae-e393-4aed-90dd-5a2d1f5c548d)



And then navigating to jubmojis page is blank?

![IMG_6839](https://github.com/jubmoji/jubmoji.quest/assets/6290822/4879ef10-9444-4881-919f-02a9f0332ddb)

Reply:
Default should be to close either collection / onboarding modal and be redirected to your `/jubmojis` page with the jubmoji you just collected being visible I think!

Reply:
Added redirect to `/jubmojis` when the modal is closed.